### ISSUE TITLE
Extend Global SCSS Stylesheet in Component

### ISSUE THREAD

**Issue by Splode** [nuxt/nuxt.js#601]
**Question**
There may be an obvious solution that I am overlooking, but I am not able to `import` my `main.scss` file from `assets`  into a component.

````scss
<style lang="scss" scoped>
@import '~assets/main.scss';
</style>
````
This generates the following error: 
````
This dependency was not found:

* !!vue-style-loader!css-loader!../node_modules/vue-loader/lib/style-compiler/index?{"id":"data-v-3bb4be04","scoped":true,"hasInlineConfig":true}!sass-loader!../node_modules/vue-loader/lib/selector?type=styles&index=0!./NavBar.vue in ./components/NavBar.vue

To install it, you can run: npm install --save !!vue-style-loader!css-loader!../node_modules/vue-loader/lib/style-compiler/index?{"id":"data-v-3bb4be04","scoped":true,"hasInlineConfig":true}!sass-loader!../node_modules/vue-loader/lib/selector?type=styles&index=0!./NavBar.vue
````

I have installed _vue-style-loader_ and added it to my config file:
````javascript
vendor: ['vue-style-loader'],
````

TLDR: **How can I import a SCSS stylesheet from `assets` into a component?**

Thanks, 
Christopher

**Comment by pi0** [comment]
@Splode I think that error is because `~` alias is not working in sass loader. Can you try to directly importing using relative paths ? `../../assets` ...

**Comment by Splode** [comment]
@pi0 Just tried using a relative path, but failed to build with same error.

**Comment by motia** [comment]
I am using sass, and ~ links works fine with nuxt. You have to make sure `node-sass` and `sass-loader` packages are installed.

PS: I don't understand what is `vendor: ['vue-style-loader']`. This property is vendor js extraction as far as I know.

**Comment by Splode** [comment]
Thanks. I did `sass-loader` installed as a _dependency_, but not as a _devDependency_. Nuxt now compiles as expected.
